### ISSUE TITLE
No primary key in oc_sociallogin_connect

### ISSUE THREAD

**Issue by tggrissom** [zorn-v/nextcloud-social-login#392]
Error from Nextcloud Log:

```
[no app in context] Error: Table "oc_sociallogin_connect" has no primary key and therefor will not behave sane in clustered setups. This will throw an exception and not be installable in a future version of Nextcloud.

POST /index.php/settings/apps/enable
from 10.10.250.42 by admin at 2023-02-04T19:24:35+00:00
```
Nextcloud Version: 25.0.3
Social-logig VersionL  5.3.0

**Comment by zorn-v** [comment]
Fixed in v5.4.0

**Comment by tggrissom** [comment]
Thanks!
